doc/rados/configuration/ceph-conf.rst

```diff
@@ -94,27 +94,26 @@ entirely.
 Skipping monitor config
 -----------------------
 
-Pass the option ``--no-mon-config`` to any process to skip the step that
-retrieves configuration information from the cluster monitors. This is useful
-in cases where configuration is managed entirely via configuration files, or
-when the monitor cluster is down and some maintenance activity needs to be
-done.
-
+The option ``--no-mon-config`` can be passed to any process in order to skip
+the step that retrieves configuration information from the cluster monitors.
+Skipping that retrieval step can be useful in cases where configuration is
```

Comment:
that/this

Reply:
I wondered about this as I was looking at it. Thanks for the benefit of your fresh eyes.

Accepted.